Sources/Avatar.swift

```diff
@@ -24,28 +24,93 @@
 
 import Foundation
 
-public struct Avatar {
-
-    public let image: UIImage?
-
-    public let highlightedImage: UIImage?
-
-    public let placeholderImage: UIImage
-
-    public init(image: UIImage? = nil, highlightedImage: UIImage? = nil, placeholderImage: UIImage) {
-        self.image = image
-        self.highlightedImage = highlightedImage
-        self.placeholderImage = placeholderImage
+open class Avatar: UIView {
```

Comment:
Ya that is what I had it called originally and then changed it for simplicity but I am not married to the idea I think I will go back to avatar view.